### PR TITLE
Add list of frontends to README

### PR DESCRIPTION
**Pull request by rjmholt**
Resolves https://github.com/PowerShell/PowerShellEditorServices/issues/969

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ PowerShell Editor Services is a PowerShell module that provides common
 functionality needed to enable a consistent and robust PowerShell development
 experience in almost any editor or integrated development environment (IDE).
 
+## Front-ends using PSES
```
Comment (rkeithhill) [inline README.md]: "Front-ends" is probably more recognizable to folks but technically these are language service client apps, right?

Comment (rjmholt) [reply README.md]: Yeah "LSP clients" is maybe better?

Comment (rkeithhill) [reply README.md]: That's a more "correct" description IMO. Whether that is meaningful to the average user who looks at this ... I dunno.

Comment (rjmholt) [reply README.md]: Well I'm hoiping that users who get to the PSES README probably understand it somewhat

**Comment by rjmholt** [comment]
I'll merge this in when the LSP page is updated, closing the issue this resolves

**Pull request merged**